Description: change gee API to 0.8
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Bug: https://github.com/Simmesimme/Gnome-Pie/issues/77
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=753863
Last-Update: 2014-07-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/CMakeLists.txt
===================================================================
--- trunk.orig/CMakeLists.txt	2013-03-17 12:15:02.000000000 +0100
+++ trunk/CMakeLists.txt	2014-07-21 17:44:36.814563290 +0200
@@ -43,7 +43,7 @@
 
 pkg_check_modules(GIO REQUIRED gio-unix-2.0)
 pkg_check_modules(CAIRO REQUIRED cairo)
-pkg_check_modules(GEE REQUIRED gee-1.0)
+pkg_check_modules(GEE REQUIRED gee-0.8)
 pkg_check_modules(X11 REQUIRED x11)
 pkg_check_modules(XML REQUIRED libxml-2.0)
 pkg_check_modules(XTST REQUIRED xtst)
@@ -155,7 +155,7 @@
 
 set(VALA_PKGS
   cairo
-  gee-1.0
+  gee-0.8
   x11
   gio-unix-2.0
   posix
